annotations/static/annotations/js/annotators/text.js

```diff
@@ -6,17 +6,58 @@
 
 var ConceptListItem = {
     props: ['concept'],
-    template: `<div class="list-group-item concept-item clearfix" id="concept-{{ concept.uri }}">
+    template: `<div>
+                <div class="list-group-item concept-item clearfix" id="concept-{{ concept.uri }}">
                    <div>
                        <a v-on:click="select" style="cursor: pointer;">{{ concept.label }} ({{ concept.authority.name }})</a>
                    </div>
                    <div class="text text-muted">{{ concept.description }}</div>
-               </div>`,
+                </div>
+
+                <div v-if="identity2" class="list-group-item concept-item clearfix" id="concept-{{ concept2 }}">
+                    <div>
+                      <a v-on:click="select" style="cursor: pointer;">{{ concept.label }} ({{ conceptPower }})</a>
+
+                  </div>
+                  <div v-if="identity2" class="text text-muted">{{ concept.description }}</div>
```

Comment:
wouldn't ```concept.label``` and ```concept.description``` have the same value as in the previous item? They should show what conceptpower actually returns.